src/style.css

```diff
@@ -1 +1,80 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+  --main-color: #00acdc;
+}
+
+body,
+nav {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  cursor: pointer;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: auto;
+  list-style: none;
+}
+
+.nav__item {
+  position: relative;
+  margin-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  line-height: 60px;
+  font-family: "Roboto", sans-serif;
```

Comment:
Move to html selector and remove from all other places.

Reply:
in this case the code stops working! Do you really need to make such changes?